README.md

```diff
@@ -147,6 +149,18 @@ It is possible to run linters for certain paths only by using [minimatch](https:
 }
 ```
 
+When mathcing, lint-staged will do the following
+
+* Resolve the git root automatically, no configuration needed.
+* Pick the staged files which are present inside the project directory.
+* Filter them using the specified glob patterns.
```

Comment:
Simplify to `Filter using minimatch` ?

Reply:
I wouldn't since in future we might want to use another matcher and will need to update the documentation.